app/controllers/users_controller.rb

```diff
@@ -5,10 +5,10 @@ def new
   end
 
   def create
-    @user = User.new(user_params)
-    if @user.save
+    @user = User.new(pending_user_params)
+    if @user.save!
       create_user
-      redirect_to profile_path
+      redirect_to  two_factor_authentication_path(@user.id)
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.